hedera-node/hedera-app/src/test/java/com/hedera/node/app/version/ServicesSoftwareVersionTest.java

```diff
@@ -63,13 +63,6 @@ void alwaysLaterThanNull() {
         assertThat(subject.compareTo(null)).isGreaterThan(0);
     }
 
-    @Test
```

Comment:
Is this test no longer required or maybe covered by other tests ?

Reply:
Since HederaSoftwareVersion is deprecated, that test is no more relevant 🙂